contracts/carbon/Strategies.sol

```diff
@@ -178,14 +178,22 @@ abstract contract Strategies is Initializable {
     // accumulated fees per token
     mapping(Token => uint256) internal _accumulatedFees;
 
+    // mapping between a pair id to its custom trading fee (in units of PPM)
+    mapping(uint128 => uint32) internal _tradingFeePPMOverrides;
```

Comment:
You can add mapping names (e.g., `mapping(uint128 pairId => uint32 fee)`) for clarity.

Reply:
Added